src/tests/test_instructions.py

```diff
@@ -84,8 +84,18 @@ def test_instruction_limit(self):
 
 
 class TestInstructionExecution:
-    @pytest.mark.skip("Needs fixing post processing and down-conversion issues.")
-    def test_measure_instructions(hw):
+    @pytest.mark.parametrize(
+        "measure_instruction",
+        [
+            "measure_mean_z",
+            "measure_mean_signal",
+            "measure_single_shot_z",
+            "measure_scope_mode",
+            "measure_single_shot_binned",
+            "measure_single_shot_signal",
+        ],
```

Comment:
For the future, it's better to use lambda's or method references than string -> getattr. Gets picked up by tools easier, refactors, fails with a meaningful message etc.

Reply:
@chemix-lunacy like so?